packages/enterprise/package.json

```diff
@@ -16,7 +16,9 @@
     "clean": "rimraf node_modules",
     "type-check": "tsc --pretty --noEmit"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
So `yarn build` was working without this.  But `yarn build:deps` in top level was not working without it.  With the error 
```
enterprise | ../back-end/types/feature.d.ts(3,55): error TS2307: Cannot find module '@growthbook/growthbook' or its corresponding type declarations.
```
shared module calls back-end/types/features.d.ts which then imports from @growthbook/growthbook.

Reply:
Another way that works is to change yarn build:deps to build enterprise only after shared finishes: `"build:deps": "wsrun -p '@growthbook/growthbook' -p '@growthbook/growthbook-react' -p 'shared' -t -c build && wsrun -p 'enterprise' -c build",`